build.sbt

```diff
@@ -60,7 +115,18 @@ val Snappy = config("snappy") extend Test
 val root = project.in(file("."))
     .configs(Integration, Benchmark, Evm, Ets, Snappy)
     .settings(commonSettings: _*)
-    .settings(libraryDependencies ++= dep)
+    .settings(
```

Comment:
I think it would be a good idea to make `dist` task dependant on `verify`. This will force to run checks before releasing a version. Something like `dist in Universal := ((dist in Universal) dependsOn verify).value`

What do you think?

Reply:
Good idea